src/main/play-doc/operation/Install.md

```diff
@@ -269,7 +313,7 @@ The [ConductR-Ansible](https://github.com/typesafehub/conductr-ansible) plays an
 
 Use create-network-ec2.yml to setup a new Virtual Private Cloud (VPC) and create your cluster in the new VPC. You only need to provide your access keys and what region to execute in. The playbook outputs a vars file for use with the build-cluster-ec.yml.
 
-The playbook build-cluster-ec2.yml launches three instances across two availability zones. ConductR is installed on all instances and configured to form a cluster. The nodes are registered with a load balancer. This playbook can be used with the newly created VPC from create-network-ec2.yml or your existing VPC and security groups.
+The playbook build-cluster-ec2.yml launches six instances across three availability zones. ConductR is installed on three instances and configured to form a cluster. ConductR Agent is installed on the remaining three instances, and configured to connect to ConductR. The ConductR Agent nodes are registered with a load balancer. This playbook can be used with the newly created VPC from create-network-ec2.yml or your existing VPC and security groups.
```

Comment:
@edwardcallahan - I have updated ConductR Ansible and AWS manual steps to cater for ConductR Agent installation.

I'd imagine we have additional 3 ConductR nodes + 3 ConductR Agent nodes (so 6 nodes in total). This is so we can update the ConductR core without taking down the agent and its bundle process at the same time. Similarly, we can upgrade the agent without the core if there's a need to do so too.

The docs around ConductR Ansible and AWS are written in advance of the actual implementation, i.e. I haven't updated the ConductR Ansible.

If you wish to update the ConductR Ansible and AWS docs only after ConductR Ansible changes are done, I'm happy to pull the changes out since they are in separate commit.